src/repositories/live.repository.ts

```diff
@@ -0,0 +1,330 @@
+import { Repository } from '../core/repository';
+import Chance = require('chance');
+import { LiveSwitchCommentsResponseRootObject } from '../responses/live.switch-comments.response';
```

Comment:
All the responses should be exported from `/src/responses/index.ts` and imported like
```ts
import {...} from '../responses'
```